homeassistant/components/bayesian/strings.json

```diff
@@ -1,7 +1,7 @@
 {
   "issues": {
     "manual_migration": {
-      "description": "The Bayesian integration now also updates the probability if the observed `to_state`, `above`, `below`, or `value_template` evaluates to `False` rather than only `True`. So it is no longer required to have duplicate, complementary entries for each binary state. Please remove the mirrored entry for `{entity}`.",
+      "description": "The Bayesian integration now also updates the probability if the observed `to_state`, `above`, `below`, or `value_template` evaluates to `False` rather than only `True`. So it is no longer required to have duplicate, complementary entries for each binary state. Please remove the mirrored entry for `{entity}` or else your calculated probabilities are likely to be incorrect.",
```

Comment:
Can you please move the strings/translations to a separate PR?
It's small and can be merged on the side.

Reply:
done